packages/mdc-dialog/README.md

```diff
@@ -0,0 +1,169 @@
+# MDC Dialog
```

Comment:
I feel like there are a few things missing here:

- Examples showing more rich content being used within dialogs, such as lists
- The dialog foundation API
- The dialog component API